pom.xml

```diff
@@ -1300,7 +1300,12 @@
             <dependency>
                 <groupId>commons-pool</groupId>
                 <artifactId>commons-pool</artifactId>
-                <version>1.6</version>
```

Comment:
it comes in as a transitive dependency in several modules from:
org.geotools:gt-referencing:jar:14.1:compile
org.geotools:gt-opengis:jar:14.1:compile
org.apache.camel:camel-netty4:jar:2.15.1:test